### DISCUSSION TITLE
Why integ testing always run in one region?

### DISCUSSION THREAD

**Discussion by zorrofox** [aws/aws-cdk#24043] [Q&A]
According to the [Running large numbers of Tests](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md#running-integration-tests) we can run parallel regions in several regions at once and even we don't specify any regions the `yarn integ-runner` will default running across `us-east-1`, `us-east-2`, & `us-west-2`. But when I run the command `yarn integ-runner` only one region will be tested and if I don't specify any region it will run on `us-east-1`.

**Comment by pahud** [comment]
It would spread failed tests into multiple regions to run in parallel like this.

![image](https://github.com/aws/aws-cdk/assets/278432/9cb6e7ea-8aaf-480c-a83b-a01212901b38)
